aikau/src/main/resources/alfresco/services/PreferenceService.js

```diff
@@ -70,8 +71,8 @@ define(["dojo/_base/declare",
        * @since 1.0.32
        */
       registerSubscriptions: function alfresco_services_PreferenceService__registerSubscriptions() {
-         this.alfSubscribe(this.getPreferenceTopic, lang.hitch(this, this.getPreference));
-         this.alfSubscribe(this.setPreferenceTopic, lang.hitch(this, this.setPreference));
+         this.alfSubscribe(topics.GET_PREFERENCE, lang.hitch(this, this.getPreference));
+         this.alfSubscribe(topics.SET_PREFERENCE, lang.hitch(this, this.setPreference));
```

Comment:
Potentially breaking change, if anyone has extended the PreferenceService, and prevents extensions modifying topics in future. That's probably fine (I can't think of a use case where changing those topics is useful), but one to call out in the release notes, perhaps?